src/python/scripts/dx-jobutil-get-identity-token

```diff
@@ -48,10 +48,10 @@ def main(aud, subject_claims):
 
     if subject_claims is not None:
```

Comment:
if there must be at least one subject claim per request, this condition should be removed and we should require at least one `--subject_claims` argument...